src/main/java/entities/User.java

```diff
@@ -55,22 +55,13 @@ public User(String password, String accountName, Profile profile,
         this.likedme = likedme;
         this.reviews = reviews;
     }
+
     // For RegularUser constructor.
     public User(String password, String accountName, Profile profile){
         super(password, accountName);
         this.profile = profile;
     }
```

Comment:
Actually, we need this overload constructor to access the database

Reply:
Please refer to the original User file before this PR. This constructor has been duplicated in lines 50 to 57. As such, we need to delete the duplicate.